### PR TITLE
Include automated_signing in signing API response

### PR DESCRIPTION
**Pull request by mstriemer**
r? @kumar303

### CODE REVIEW

**Comment by kumar303** [comment]
I dislike this field name but it would be worse to call it something other than the real model field name :)

Can you adjust the docs? http://olympia.readthedocs.org/en/latest/topics/api/signing.html#checking-the-status-of-your-upload

r+wc

**Comment by mstriemer** [comment]
I removed the bit in the docs about needing to pass tests if unlisted also.

File: docs/topics/api/signing.rst [hunk 1]
```diff
@@ -131,6 +132,9 @@ property.
             }
 
     :>json active: version is active.
+    :>json automated_signing:
+        if the version will be signed automatically, if this is false it will
```
Comment (kumar303) [inline docs/topics/api/signing.rst]: these read better to me as two sentences so no comma after automatically

Comment (kumar303) [inline docs/topics/api/signing.rst]: I know the other fields use incomplete sentences but I think you should make it complete, like: If true, the version will be signed automatically.

File: docs/topics/api/signing.rst [hunk 2]
```diff
@@ -86,12 +86,12 @@ passed using the ``valid`` property and check the results with
...
-then it will be reviewed by a human and that will take a bit
-longer. Once review is complete then the ``reviewed`` property
-will be set and you can check the results with the ``passed_review``
-property.
+In the case of unlisted add-ons this will happen automatically.  If your add-on
+is listed then it will be reviewed by a human and that will take a bit longer.
+You can check the ``automated_signing`` property to see if signing will happen
+automatically or after a manual review. Once review is complete then the
+``reviewed`` property will be set and you can check the results with the
+``passed_review`` property.
```
Comment (kumar303) [inline docs/topics/api/signing.rst]: oh yeah, forgot about this. Thanks.

**Comment by kumar303** [comment]
r+wc

File: docs/topics/api/signing.rst (hunk 1 above)
Comment (mstriemer) [inline docs/topics/api/signing.rst]: Yeah the incomplete sentences were tripping me up. That's why I used the comma.

**Pull request merged**